src/main/java/uk/ac/ebi/intact/app/internal/io/HttpUtils.java

```diff
@@ -80,32 +128,31 @@ public static JsonNode postJSON(String url, Map<Object, Object> data, Manager ma
         }
     }
 
-    public static JsonNode postJSON(String url, Map<Object, Object> data, Manager manager, Supplier<Boolean> isCancelled) {
+    public static JsonNode postJSON(String url, Map<Object, Object> body, Manager manager, Supplier<Boolean> isCancelled) {
         try {
             HttpRequest request = HttpRequest.newBuilder()
-                    .POST(buildFormDataFromMap(data))
                     .uri(URI.create(url))
+                    .POST(HttpRequest.BodyPublishers.ofString(mapper.writeValueAsString(body)))
```

Comment:
I created a new endpoint in the GraphDB-WS, and it is deployed now. Endpoints is to replace `network/data` with a proper request body. It is named `network/fromInteractors` to be consistent with the other one.

I haven't tested this, so we should test it before this is merged.